apps/meteor/package.json

```diff
@@ -261,7 +261,7 @@
 		"fibers": "^5.0.1",
 		"file-type": "^16.5.3",
 		"filenamify": "^4.3.0",
-		"filesize": "^3.6.1",
+		"filesize": "^8.0.7",
```

Comment:
why is that important for this pull request?

Reply:
That's because filesize doesn't expose any typings for older versions. And the @types package was not working properly.

Good thing is that, even with the update, our usage of the lib is very basic across RC (we just pass a byte amount and get a string rep) and that API is still there even with the 5 majors updates.